borrowing/serializers.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework import serializers
+from borrowing.models import Borrowing
+
+
+class BorrowingSerializer(serializers.ModelSerializer):
```

Comment:
Can you please rename to BorrowingCreateSerializer?

Reply:
Sure